monkey/infection_monkey/monkey.py

```diff
@@ -177,6 +178,7 @@ def __init__(self, args, ipc_logger_queue: multiprocessing.Queue, log_path: Path
         self._heart.start()
 
         self._current_depth = self._opts.depth
+        self._servers: Sequence[str] = None
```

Comment:
Doesn't this raise a mypy warning? Are your pre-commit hooks running?

What's the purpose of this change?

Reply:
We need to have `self._servers` for the `AgentCommandBuilderFactory` so I defaulted it to None at first. Maybe I should do empty list or nothing at all ? 

The pre-commits are running but there are quite few for the monkey.py and maybe I missed it.

Reply:
2 options:

1. `servers` is already passed to `self._build_master()`. `self._build_master()` calls `self._build_puppet()`, where `AgentCommandBuilderFactory` is constructed. Just pass `servers` to `self._build_puppet()`.
2. There's no need to define this here. Just define it where it's initialized.